homeassistant/components/cover/homematic.py

```diff
@@ -69,3 +71,39 @@ def _init_data_struct(self):
         """Generate a data dictoinary (self._data) from metadata."""
         self._state = "LEVEL"
         self._data.update({self._state: STATE_UNKNOWN})
+        try:
+            self._data.update(
+                {'LEVEL_2': self._hmdevice.get_cover_tilt_position()})
+        except:
+            pass
```

Comment:
It will be self updated. Look into the NODE to check if the `LEVEL_2` exists and put it into the data dict with STATE_UNKNOWN.